airflow/contrib/operators/s3_to_gcs_operator.py

```diff
@@ -201,7 +203,7 @@ def execute(self, context):
     # Following functionality may be better suited in
     # airflow/contrib/hooks/gcs_hook.py
     @staticmethod
-    def _gcs_object_is_directory(object):
-        bucket, blob = _parse_gcs_url(object)
+    def _gcs_object_is_directory(object):  # pylint:disable=redefined-builtin
```

Comment:
Is it required?

Reply:
I can refactor the argument and use `obj` for example